app/uk/gov/hmrc/excisemovementcontrolsystemapi/utils/DateTimeService.scala

```diff
@@ -18,19 +18,26 @@ package uk.gov.hmrc.excisemovementcontrolsystemapi.utils
 
 import com.google.inject.{ImplementedBy, Singleton}
 
-import java.time.Instant
-import java.time.temporal.ChronoUnit
+import java.time.format.DateTimeFormatter
+import java.time.{Instant, ZoneOffset, ZonedDateTime}
 
 @ImplementedBy(classOf[DateTimeServiceImpl])
 trait DateTimeService {
   def timestamp(): Instant
-
-  def timestampToMilliseconds(): Instant
```

Comment:
I have removed this  and used the timestamp every where like before, because we only need milliseconds in the EIS header that take dateTime as a string. So I only format the datetime when we pass it to the headers. See what you think.

Reply:
It is fine for the timestamp to be stored in the Db in it's raw format but whenever we display a timestamp back to the users or ask them to submit a timestamp (updatedSince query parameter for example), we should be using a consistent display format throughout. Can we make sure that all display timestamps are displayed to 3DP with Z on the end?